README.md

```diff
@@ -8,6 +8,15 @@
 
 This module implements Event Streams with topics, partitions, throughput, storage size, cleanup policy, retention time, retention size, segment size and schema.
 
+## Note for KMS encryption
+
+The Event Streams service supports payload data encryption using a root key CRN of a Key Management Services like Key Protect or Hyper Protect Crypto Services (HPCS), see https://cloud.ibm.com/docs/EventStreams?topic=EventStreams-managing_encryption
+The root key CRN can be specified using the `kms_key_crn` variable.
+The authorization policy to allow the Event Streams service to access the key management service instance as a Reader MUST be configured in advance, see https://cloud.ibm.com/docs/account?topic=account-serviceauth
+This cannot be managed in the same Terraform state as the Event Streams service instance, because on destroy the instance is soft deleted to allow for recovery.
+The authorization policy must still be inplace when the instance is hard deleted or restored, otherwise the de-registration of the instance from the root key will fail on the backend, and you will not be able to delete the root key without creating a support case,
+for more details see https://cloud.ibm.com/docs/EventStreams?topic=EventStreams-managing_encryption#using_encryption
```

Comment:
You would need blank lines if you wanted to paragraph that text. It all shows in same paragraph:
![image](https://github.com/terraform-ibm-modules/terraform-ibm-event-streams/assets/29608224/57d88a3a-d280-45bc-a7ef-0376c33fb77c)

Perhaps use hyperlinks too so its easier on the eyes